scripts/zones/RoMaeve/Zone.lua

```diff
@@ -62,64 +50,38 @@ function onZoneIn(player,prevZone)
     return cs;
 end;
 
------------------------------------
--- onRegionEnter
------------------------------------
-
 function onRegionEnter(player,region)
 end;
 
------------------------------------
--- onGameDay
------------------------------------
-
 function onGameDay()
-
-    -- Full moon + "clear" weather stuff (actually "sunshine" weather, widespread misconception since Ro'Maeve does not have "clear" weather ever)
-    local Moongate_Offset = 17277195; -- _3e0 in npc_list
-    local hour = VanadielHour();
-
-    if (IsMoonFull() == true and GetNPCByID(Moongate_Offset):getWeather() == WEATHER_SUNSHINE) then
-        GetNPCByID(Moongate_Offset):openDoor(432); -- 3 game hours worth of seconds
-        GetNPCByID(Moongate_Offset+1):openDoor(432);
-        GetNPCByID(Moongate_Offset+7):openDoor(432); -- visual part of Qu'Hau Spring
+    local MOONGATE_OFFSET = 17277195; -- _3e0
```

Comment:
tfw wren is removing all these

Reply:
I can move it to a NPCIDs if you prefer, it was just a quick fix to what was there.

Reply:
wrens been putting the NPC values in the mobIDs.lua, which I intend to rename to enitityIDs.lua for semantics sake when I take care of assaults "ids.lua"

Reply:
I actually initially added it to MobIDs, then backtracked as I didn't think having conflicting entity types was a great idea.

I can throw it in MobIDs for now if that is what your renaming plan is.